src/pkg/packaging/rpm/dotnet-hostfxr-rpm_config.json

```diff
@@ -0,0 +1,43 @@
+{
+  "maintainer_name": "Microsoft",
+  "maintainer_email": "dotnetcore@microsoft.com",
+  
+  "package_name": "%HOSTFXR_RPM_PACKAGE_NAME%",
+  "install_root": "/usr/share/dotnet",
```

Comment:
Just as an FYI, the FHS says:

> The /usr/share hierarchy is for all read-only architecture independent data files.

The `dotnet` application, CoreCLR and various other components are architecture dependent. They should be in `/usr/lib64/` (or `/usr/lib/`, or whatever the equivalent in the distribution is).